databricks/koalas/indexes.py

```diff
@@ -145,6 +145,21 @@ def to_pandas(self) -> pd.Index:
 
     toPandas = to_pandas
 
+    def to_numpy(self, dtype=None, copy=False):
+        """
+        A NumPy ndarray representing the values in this Index or MultiIndex.
```

Comment:
Can you add a note:

```
        .. note:: This method should only be used if the resulting NumPy ndarray is expected
            to be small, as all the data is loaded into the driver's memory.
```

Also, can you please port related docs (e.g., Parameters, Returns) to here? (see https://github.com/pandas-dev/pandas/blob/v0.25.3/pandas/core/base.py#L916-L1010)

Reply:
We should also remove https://github.com/databricks/koalas/blob/master/databricks/koalas/missing/indexes.py#L100 and https://github.com/databricks/koalas/blob/master/databricks/koalas/missing/indexes.py#L100

Also, add this new API into https://github.com/databricks/koalas/blob/master/docs/source/reference/indexing.rst

Reply:
Thanks!  Will do.